src/pages/Climate.tsx

```diff
@@ -63,15 +63,7 @@ const Climate: React.FC = () => {
     return (
         <IonPage>
             <IonHeader>
-                {/*
-                This toolbar is the at the tope of the modal and displays the name of the current
-                tab being views, i.e. Climate
-                */}
-                <IonToolbar color="#D99648" className="ClimateToolBar">
-                    <IonTitle size="large" color="white">
-                        Climate
-                    </IonTitle>
-                </IonToolbar>
+                <IonToolbar></IonToolbar>
```

Comment:
I would get rid of the Toolbar entirely from here since we've already taken care of it in the other component